framework/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Yii Framework 2 Change Log
 
 2.0.50 under development
 ------------------------
+- Bug #20045: Fix type `boolean` in `MySQL` (terabytesoftw)
```

Comment:
We have this line twice now?

Reply:
![image](https://github.com/yiisoft/yii2/assets/42547589/da4aeee3-7727-447d-aece-a32241291ffd)

Nope, MSSQL and MySQL,

Reply:
I'm blind apparently :) sorry

Reply:
no problem.